flink-kubernetes/src/test/java/org/apache/flink/kubernetes/kubeclient/Fabric8FlinkKubeClientTest.java

```diff
@@ -454,6 +451,22 @@ void testWatchPodsAndDoCallback() throws Exception {
         assertThat(podModifiedAction.get()).isEqualTo(Action.MODIFIED);
     }
 
+    @Test
+    void testWatchPodsAndDoCallbackFail() throws Exception {
+        mockPodEventWithLabelsFail(
+                NAMESPACE, TASKMANAGER_POD_NAME, KUBERNETES_ZERO_RESOURCE_VERSION, TESTING_LABELS);
+        TestingWatchCallbackHandler<KubernetesPod> watchCallbackHandler =
+                TestingWatchCallbackHandler.<KubernetesPod>builder().build();
+        Long watchStartTime = System.currentTimeMillis();
```

Comment:
I'd suggest to use `System.nanoTime()`.

`System.currentTimeMillis()` can be updated at a larger granularity and is not suitable ms-level time counting.